lib/kb_staging_exporter/Utils/staging_downloader.py

```diff
@@ -335,8 +342,20 @@ def export_to_staging(self, ctx, params):
         self._mkdir_p(staging_dir)
         files = os.listdir(result_dir)
         for file in files:
+            # Doesn't use move to keep result dir around for other apps that might not
+            # have access to the staging ara
             shutil.copy2(os.path.join(result_dir, file), staging_dir)
 
+        # This is a KBase specific hack to allow the staging service to delete the files and
+        # folders written by this module. Currently the staging service runs as user 800 and
+        # this module runs as root (bleah) so the staging service throws an error if the user
+        # tries to delete the folder. The staging service belongs to the root group, however,
+        # so if we add write privs to the root group that solves the issue.
+        # Longer term this app should not run as root and should chown ownership to the staging
+        # service when it has a static user name vs. a number that might change.
+        basemode = os.stat(staging_dir).st_mode
+        self._recursive_chmod(staging_dir, basemode | stat.S_IWGRP)
```

Comment:
okay. So that was just combining current mode with additional write permission? 

LGTM 👍

Reply:
right, exactly